persistence/src/main/kotlin/de/cyface/persistence/DatabaseMigrator.kt

```diff
@@ -92,6 +92,17 @@ class DatabaseMigrator(val context: Context) {
     val MIGRATION_9_10: Migration = migrationFrom9To10()
 
     companion object {
+        /**
+         * Adds the [de.cyface.persistence.model.Measurement.filesSize] column.
+         */
+        val MIGRATION_19_20 = object : Migration(19, 20) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                // Add the new filesSize column with default value 0 for existing rows
+                // This should be fine as we don't upload attachments in production, yet.
+                database.execSQL("ALTER TABLE Measurement ADD COLUMN filesSize INTEGER NOT NULL DEFAULT 0")
+            }
+        }
+
```

Comment:
As we don't use the object store attachments upload on production yet, we should be fine not don't have to calculate the files size for measurements collected with the previous version here but just use "0" instead.